argilla/docs/tutorials/image_classification.ipynb

```diff
@@ -11,8 +11,8 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    client.datasets("image_classification_dataset").delete()
</pre><p>I think we should remove this, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>